StyleGuide.md

```diff
@@ -821,6 +821,16 @@ Highlight shell and SQL commands where appropriate using the following info:
 
 Start shell code samples with `~~~ shell` followed by a line break. The first character of the next line must be the terminal marker `$`. For multi-line shell commands, use a backslash (`\`) at the end of each line to indicate a line break.
 
+In an example command to download an artifact using the command line, `curl` is preferred over `wget` because it is included in Linux, macOS, and recent builds of Windows. Use syntax like:
+
+```
+curl -o {optional_output_path}/{output_file_name} [-H {header_keys_and_values}] {url} 
+```
+
+- If you do not specify `-o` or `--output`, the URL's contents are printed to standard output, even if it is a binary file. This can corrupt your terminal session and is useful only when piping or redirecting the output to a subsequent command.
+- If you do not include a path in your output file name, the file is saved to the current working directory.
+- Headers are optional, but certain APIs such as the CockroachDB Cloud API require  certain header fields to be set.
```

Comment:
Neither. I was taking into consideration the different types of `wget` examples I updated to use `curl` and to show what to do when their behavior differs, but I wanted to keep it very narrow to the sorts of ways our docs use `curl` rather than exhaustively document all the options to each command, and to try to prevent you from streaming the download binary to standard output.

- We had several `wget` examples that specify an output file name, path, or both.
- We had several examples where passing extra headers is a key part of the example, so I wanted to show how you'd do that with `curl`.

Part of the motivation for why I did it this way is that this is very specific info about one specific type of code example, and there isn't a great home for it in the style guide's TOC. I was hoping the info would come up in searches. WDYT?